notebooks/02 - Custom layers.ipynb

```diff
@@ -0,0 +1,256 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "71c77df9-1d42-4993-9a0a-d311064d03c9",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from unfolded.map_sdk import UnfoldedMap\n",
+    "unfolded_map = UnfoldedMap(\n",
+    "   # mapUrl='http://localhost:8080/incognito'\n",
+    "   # mapUrl='https://staging.unfolded.ai/incognito'\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "67fbd3d3-2eb6-45f4-b3e4-7391fc5282a7",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from sidecar import Sidecar\n",
+    "sc = Sidecar(title='Unfolded map demo', anchor='split-right')\n",
+    "with sc:\n",
+    "    display(unfolded_map)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "05b6f62b",
+   "metadata": {},
+   "source": [
+    "## Load data into a DataFrame "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "3366a68b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from uuid import uuid4\n",
+    "dataset_id=uuid4()"
```

Comment:
This `dataset_id` can be anything right? This is just for demo purposes?

Reply:
Yes, it has to be a UUID or it will throw 
![image](https://user-images.githubusercontent.com/351828/121353689-6976bd00-c92e-11eb-8442-771018cc4ae1.png)
We don't have this check in JS btw, maybe we should.

Reply:
In the docs we say:
`uuid` – Unique identifier of the dataset (will be auto-generated in Jupyter when not provided)
this is still true? it will be auto-generated?

Reply:
Yes, it can be omitted, then it will be auto-generated. You need it if you want to update/delete a previously added dataset or to add a layer. But it's not super easy to get it back when it's autogenerated. So it's easier to generate manually.

Reply:
Good point. Might be good to mention this somewhere in the docs?